target/linux/ramips/base-files/etc/board.d/02_network

```diff
@@ -78,6 +78,7 @@ ramips_setup_interfaces()
 	dir-320-b1|\
 	dir-610-a1|\
 	dir-615-h1|\
+	gb-pc1|\
```

Comment:
Sorry about that one.
Can you please fix that one ... my github is out of wack.
I think it related to being in China and behind the firewall.

I'll start on the MT7621 "compatible strings" tonight.

And Thanks!